adobe-esign-connector/src/com/axonivy/connector/adobe/esign/connector/service/AdobeSignService.java

```diff
@@ -103,6 +118,23 @@ public String uploadDocumentAndCreateSimpleAgreementWithFormFields(String name,
 
 		return agreementId;
 	}
+	
+	/**
+	 * Convenience method to call upload document and create agreement methods
+	 *
+	 * @param name
+	 * @param workflowId
+	 * @param signerEmail
+	 * @param upload
+	 * @return
+	 */
+	public String uploadDocumentAndCreateSimpleAgreementWithFormFields(String name, String signerEmail, List<UploadWrapper> upload, List<AgreementsFormFieldGenerators> formFieldGenerators) {
+		List<String> documentIds = uploadDocuments(upload);
+		String agreementId = createAgreement(
+				buildSimpleAgreementWithFormFields(name, documentIds, signerEmail, formFieldGenerators));
+
+		return agreementId;
```

Comment:
can we return the `createAgreement(buildSimpleAgreementWithFormFields(name, documentIds, signerEmail, formFieldGenerators));` instead of create a local var then returns it?

Reply:
Hi @nqhoan-axonivy , thanks for the feedback, should be addressed already 😄